k9mail/src/main/java/com/fsck/k9/planck/ui/fragments/AccountSetupBasicsFragment.kt

```diff
@@ -127,14 +128,6 @@ class AccountSetupBasicsFragment : AccountSetupBasicsFragmentBase() {
         inflater.inflate(R.menu.account_setup_basic_option, menu)
     }
 
-    private fun setupToolbar() {
```

Comment:
is this a function of the parent class? I see you still call it with params, even if it has been removed

Reply:
nevermind, found it